### DISCUSSION TITLE
fraud alert module

### DISCUSSION THREAD

**Discussion by jester223** [DSJAS/DSJAS#13] [Ideas]
i noticed that the felocity bank there is a fraud alert that prevents transfers. I think it would be nice if we could have a module that work something like that would pop up no matter what theme for the bank you are using. just a thought tho

**Comment by SinisterSpatula** [comment]
Yeah that's a nice idea.  If I can figure out how, I'd definitely create that.  I also want to make "OTP" fake 2-factor authentication module.

**Comment by ejv2** [comment]
In terms of working no matter what theme you are using, that shouldn't be an issue for themes, as modules are designed *specifically* to work regardless of the theme.

Right now, some of the theme-module compatibility is a bit lacking in themes other than the default one, as I am yet to draw up a fully standardised set of endpoints for module insertions. But, this doesn't sound like a situation that would be too badly affected by this.

The fake OTP module may work in a very similar way to the frustrating login module. Perhaps [take a look at that](https://github.com/DSJAS/DSJAS/tree/master/public/admin/site/modules/frustrating_login). If you create a fork and add create some of the modules you want adding, I am more than happy to accept contributions.

Right now, my personal focus is on the statistics system and getting that merged before the next latest release.

**Comment by ejv2** [comment]
As of v0.1.3-alpha, attempting to perform a transaction on a disabled account will show a fraud alert in all the default themes. An error code is delivered via the API so custom themes can also display this error.
